src/sinks/console/sink.rs

```diff
@@ -27,40 +42,55 @@ where
     T: io::AsyncWrite + Send + Sync + Unpin,
 {
     async fn run(mut self: Box<Self>, mut input: BoxStream<'_, Event>) -> Result<(), ()> {
-        while let Some(event) = input.next().await {
+        if matches!(
+            (self.encoder.serializer(), self.encoder.framer()),
+            (
+                codecs::encoding::Serializer::Json(_),
+                Framer::CharacterDelimited(CharacterDelimitedEncoder { delimiter: b',' }),
+            )
+        ) {
+            self.write_all(b"[").await?;
```

Comment:
This is a little odd. Are we sure it's worthwhile to support JSON vs just newline-delimited JSON for a streaming sink like this?

Reply:
> This is a little odd. Are we sure it's worthwhile to support JSON vs just newline-delimited JSON for a streaming sink like this?

I agree there. However, we currently don't explicitly disallow users from configuring the framing method to `character_delimited`, so they could still configure that if they really wanted

Reply:
> Yeah, agreed, this sink should only use ndjson not json arrays.

For what it's worth, even when setting the `json` `StandardsEncoding`, this sink also uses `newline_delimited` framing in this PR